testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_database.py

```diff
@@ -362,3 +368,404 @@ def test_copy_prebuilt_database(params_from_base_test_setup, encrypted):
+    base_url = params_from_base_test_setup["base_url"]
+    sg_db = "db"
+    sg_admin_url = params_from_base_test_setup["sg_admin_url"]
+    sg_blip_url = params_from_base_test_setup["target_url"]
+
+    if liteserv_version < "2.8.0":
+        pytest.skip('This test supports for a feature from hydrogen(2.8.0)')
+
+    # reset SGW configuration
+    c = cluster.Cluster(config=cluster_config)
+    c.reset(sg_config_path=sg_config)
+
+    # 1a. create a db, and add document to the cbl db
+    db = Database(base_url)
+    db_name = "test_close_db_" + str(time.time())
+    log_info("Creating a Database {} at test setup".format(db_name))
+    db_config = db.configure()
+    cbl_db = db.create(db_name, db_config)
+
+    num_of_docs = 100
+    channel1 = ["Replication-1"]
+    channel2 = ["Replication-2"]
+    channel3 = ["Replication-3"]
+
+    db.create_bulk_docs(num_of_docs, "ch1", db=cbl_db, channels=channel1)
+    db.create_bulk_docs(num_of_docs, "ch2", db=cbl_db, channels=channel2)
+    db.create_bulk_docs(num_of_docs, "ch3", db=cbl_db, channels=channel3)
+
+    # 1b. prepare SGW, create 3 users on 3 different channels and set up connections
+    username1 = "autotest"
+    username2 = "autotest2"
+    username3 = "autotest3"
+    password = "password"
+
+    sg_client = MobileRestClient()
+    sg_client.create_user(sg_admin_url, sg_db, username1, password=password, channels=channel1)
+    cookie1, session_id1 = sg_client.create_session(sg_admin_url, sg_db, username1)
+
+    sg_client.create_user(sg_admin_url, sg_db, username2, password=password, channels=channel2)
+    cookie2, session_id2 = sg_client.create_session(sg_admin_url, sg_db, username2)
+
+    sg_client.create_user(sg_admin_url, sg_db, username3, password=password, channels=channel3)
+    cookie3, session_id3 = sg_client.create_session(sg_admin_url, sg_db, username3)
+
+    # 2. start 3 replicators on 3 different channels, set all continous to True
+    replicator = Replication(base_url)
+    authenticator = Authenticator(base_url)
+
+    replicator_authenticator1 = authenticator.authentication(session_id1, cookie1, authentication_type="session")
+    repl_config1 = replicator.configure(source_db=cbl_db, replicator_authenticator=replicator_authenticator1, target_url=sg_blip_url,
+                                        replication_type="push_pull", continuous=True, channels=channel1)
+
+    replicator_authenticator2 = authenticator.authentication(session_id2, cookie2, authentication_type="session")
+    repl_config2 = replicator.configure(source_db=cbl_db, replicator_authenticator=replicator_authenticator2, target_url=sg_blip_url,
+                                        replication_type="push", continuous=True, channels=channel2)
+    replicator_authenticator3 = authenticator.authentication(session_id3, cookie3, authentication_type="session")
+    repl_config3 = replicator.configure(source_db=cbl_db, replicator_authenticator=replicator_authenticator3, target_url=sg_blip_url,
```

Comment:
is it possible to use configure_and_replicate method  rather than having separate calls for configure, create, start, wait for replication  which combines all 4 methods and will have few steps?